egs/ksponspeech/ASR/RESULTS.md

```diff
@@ -0,0 +1,68 @@
+## Results
+
+### Streaming Zipformer-Transducer (Pruned Stateless Transducer + Streaming Zipformer)
+
+#### [pruned_transducer_stateless7_streaming](./pruned_transducer_stateless7_streaming)
+
+Number of model parameters: 79,022,891, i.e., 79.02 M
+
+##### Training on KsponSpeech (with MUSAN)
+
```

Comment:
Could you upload the models to huggingface and put their links here?

Reply:
> Could you upload the models to huggingface and put their links here?

I uploaded the models to huggingface. See [this link](https://huggingface.co/johnBamma/icefall-asr-ksponspeech-pruned-transducer-stateless7-streaming-2024-06-12)

Fix it in 3c970e7